src/pipx/main.py

```diff
@@ -183,6 +183,18 @@ def package_is_url(package: str, raise_error: bool = True) -> bool:
     return False
 
 
+def package_is_path(package: str):
+    if os.path.sep in package:
+        raise PipxError(
+            pipx_wrap(
+                f"""
+                Error: Path '{package}' given as package.
```

Comment:
We can't determine it fully this way, so probably a more cautious error would be good.

```suggestion
                Error: '{package}' looks like a path.
```